tests/publish/ninjs_formatter_test.py

```diff
@@ -311,7 +311,7 @@ def test_item_with_usable_associations(self):
         formatted = json.loads(doc)
         self.assertIn('associations', formatted)
         self.assertIn('image', formatted['associations'])
-        image = formatted['associations']['image']
+        image = formatted['associations']['image'][0]
```

Comment:
Initially I implemented the structure according to the standard. Later @petrjasek changed it: from dict values to list of dict values because it was easier to transform from the item group structure. So currently our (not fully ninjs compatible) structure has lists of dicts values for associations dict. If we want to change this structure, that is another topic for another PR.